The power that a wind turbine delivers is proportional to the cube of the wind velocity.

$$P_D = \frac{1}{2}\dot{m}v^2 $$

where $\dot{m}$ is the mass flow rate of the air and $v$ is the velocity of the air. We can re-write this equation by recalling that 

$$\dot{m} = \rho \dot{V} = \rho A v $$

Then:

$$P_D = \frac{1}{2}(\rho A v)v^2 $$

$$P_D = \frac{1}{2}\rho A v^3 $$

The actual power density from the wind is the product of the pressure on the surface, $p$, and the velocity, $w$, with which the surface drifts downwind. The wind pressure is:

$$p = \frac{1}{2} \rho C_D  (v - w)^2 $$

where $C_D$ is the drag coefficient and the power density, $P_D$, is then:

$$P_D = p w = \frac{1}{2} \rho C_D  (v - w)^2 w $$

We can maximize this value by taking the partial derivative and setting it to zero:

$$P_D = \frac{1}{2} \rho C_D (v^2w + w^3 - 2vw^2 ) $$

$$\frac{\partial P_D}{\partial w} = \frac{1}{2} \rho C_D (v^2 + 3w^2 - 4vw ) $$

$$\frac{\partial P_D}{\partial w} = \frac{1}{2} \rho C_D (v^2 + 3w^2 - 4vw ) = 0$$

$$\frac{\partial P_D}{\partial w} = (v^2 + 3w^2 - 4vw ) = 0$$

If $w = v/3$, we see that:

$$(v^2 + 3(v/3)^2 - 4v(v/3) ) = 0$$

$$v^2 + (1/3)(v^2) - v^2(4/3)  = 0$$

$$v^2  - v^2(3/3)  = 0$$

$$0 = 0$$

So, the maximum value of $P_D$ is then:

$$P_\text{max} = \frac{1}{2} \rho C_D  (v - (v/3))^2 (v/3) $$

$$P_\text{max} = \frac{1}{2} \rho C_D  ((2/3)^2v^2) (v/3) $$

$$P_\text{max} = \frac{1}{2} \rho C_D  ((4/27)v^3) $$

$$P_\text{max} = \frac{2}{27}\rho C_D  v^3 $$

The ratio of $P_\text{max}$ to the wind power, $P_w$, is:

$$ \frac{P_\text{max}}{P_w} = \frac{\frac{2}{27}\rho C_D  v^3}{\frac{1}{2}\rho v^3 } = \frac{4}{27} C_D $$

The drag coefficient of a large flat plate at low subsonic velocities is 

$$C_D = 1.28 $$

So in this case at best it would be possible to extract this amount of energy from the wind:

In [1]:
CD = 1.28
Pratio = (4/27)*CD; print(Pratio)

0.18962962962962962


If we have the situation where any molecule that strikes a surface is reflected back against the wind without interfering with the incoming molecules, we can achieve:

$$p = 2 \rho v^2 $$

Recalling our definition of the real pressure being equal to the dynamic pressure multiplied by the drag coefficient:

$$p = \frac{1}{2}\rho v^2 C_D $$

Substituting yields:

$$ 2 \rho v^2 = \frac{1}{2}\rho v^2 C_D $$

$$ 4 =  C_D $$

In [2]:
CD_max = 4
Pratio_max = (4/27)*CD_max; print(Pratio_max)

0.5925925925925926


If $C_D = 4$, this means that $p = 2 \rho v^2$. Recalling that:

$$P_\text{max} = \frac{2}{27}\rho C_D  v^3 $$

$$P_\text{max} = \frac{2}{27}\rho 4  v^3 $$

$$P_\text{max} = \frac{8}{27}\rho v^3 $$

$$P_\text{max} = \frac{16}{27} \frac{1}{2} \rho v^3 $$

The factor of $\frac{16}{27}$ is known as the Betz limit or Betz efficiency. That is, it is only possible to extract up to 59.3 \% of the ``power in the wind''.